cypress/e2e/po/pages/extensions.po.ts

```diff
@@ -37,6 +38,10 @@ export default class ExtensionsPagePo extends PagePo {
     return this.self().get('.data-loading');
   }
 
+  waitForTabs() {
+    return cy.get('[data-testid="extension-tabs"]', LONG_TIMEOUT_OPT).scrollIntoView().should('be.visible');
```

Comment:
This exists in the root `component` po
```
  checkVisible(): Cypress.Chainable<boolean> {
    return this.self().scrollIntoView().should('be.visible');
  }
```
The extension po also has an instance of `Tabbed` po, so this could just be
```
return this.extensionTabs.checkVisible()
```

Reply:
@richard-cox I don't think it can - I need to supply the longer timeout, which can be done with the `get` but is not done by `checkVisible` and can not easily, since this uses `self` which has already processed the get and we can not easily add a timeout to that.

Reply:
Good point. This may be a good time to start wiring this in, i can foresee a lot of copy and paste in the future. Something like this in component.po.ts should work
```
type getOptions = Partial<Loggable & Timeoutable & Withinable & Shadow>;
...
export default class ComponentPo {
  public self: (options?: getOptions) => CypressChainable;
...
  constructor(...args: Array<any>) {
    if (typeof args[0] === 'string') {
      const [selector, parent] = args as [string, CypressChainable];

      this.self = (options) => (parent || cy).get(selector, options);
...
  checkVisible(options?: getOptions): Cypress.Chainable<boolean> {
    return this.self(options).scrollIntoView().should('be.visible');
  }

```

then used as 
```
this.extensionTabs.checkVisible({ timeout: LONG_TIMEOUT_OPT})
```

Other option could be to below, but it would apply to all usages of self
```
new TabbedPo(cy.get('[data-testid="extension-tabs"]', LONG_TIMEOUT_OPT))
```

Reply:
As discussed separately, if there's improvements coming in another PR this is ok as is

Reply:
I've taken the suggestion and updated as discussed